jina/orchestrate/flow/base.py

```diff
@@ -2817,6 +2806,8 @@ def _update_network_interface(self, **kwargs):
         :param kwargs: new network settings
         """
         self._gateway_kwargs.update(kwargs)
+        # reset client
+        self._client = None
```

Comment:
maybe better to reset the client on flow start/stop ?

Reply:
The client will reset on next usage because a new client is created if the self._client is None.